commit 42e5386d1c363be058ea4186351bef721539b592
Author: Greg Kroah-Hartman <gregkh@suse.de>
Date:   Tue Oct 5 09:58:02 2010 -0700

    Staging: brcm80211: typedefs.h: remove NULL definition
    
    No need for a driver to define NULL, the core kernel handles that.
    
    Cc: Brett Rudley <brudley@broadcom.com>
    Cc: Henry Ptasinski <henryp@broadcom.com>
    Cc: Nohee Ko <noheek@broadcom.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

--- a/drivers/staging/brcm80211/include/typedefs.h
+++ b/drivers/staging/brcm80211/include/typedefs.h
@@ -96,10 +96,6 @@ typedef signed int int32;
 #define TRUE	1		/* TRUE */
 #endif
 
-#ifndef NULL
-#define	NULL	0
-#endif
-
 #ifndef OFF
 #define	OFF	0
 #endif
